onadata/libs/tests/utils/test_csv_import.py

```diff
@@ -82,11 +87,13 @@ def test_submit_csv_and_rollback(self):
 
         count = Instance.objects.count()
         csv_import.submit_csv(self.user.username, self.xform, self.good_csv)
-        self.assertEqual(Instance.objects.count(),
-                         count + 9, u'submit_csv test Failed!')
+        self.assertEqual(Instance.objects.count(), count + 9,
+                         u'submit_csv test Failed!')
         # Check that correct # of submissions belong to our user
-        self.assertEqual(Instance.objects.filter(user=self.user).count(),
-                         count + 8, u'submit_csv username check failed!')
+        self.assertEqual(
+            Instance.objects.filter(user=self.user).count(),
+            count + 8,
```

Comment:
d1607c5cdaa376b8000933e588e534bf7b583ba8, f9bafa8a7fa6208b6a1d7c19891cbeb65ef65855, 14b544db3aec5dc470c5ba076d87178824cf36b9 and f47b864b83db956adff7f2fbb4f7770f375cd9fe are the key commits that are relevant in tackling #914.